modules/nf-core/ataqv/ataqv/main.nf

```diff
@@ -50,4 +50,17 @@ process ATAQV_ATAQV {
         ataqv: \$( ataqv --version )
     END_VERSIONS
     """
+
+    stub:
+    def prefix        = task.ext.prefix ?: "${meta.id}"
+    def args          = task.ext.args   ?: ''
+    def problems_cmd  = args.contains("--log-problematic-reads") ? "echo stub > 1.problems" : ""
+    """
+    echo stub > ${prefix}.ataqv.json
```

Comment:
Normally we just `touch` the files that don't need gzipping.

Reply:
You're totally right. It's just that I had some trouble with md5sum inconsistency but it was just because I set `options "stub"` instead of `options "-stub"`